scss/core/elements/_input.datepicker.scss

```diff
@@ -111,15 +100,45 @@
 		}
 
 		luid-daterange-picker {
-			.lui.tagged.input {
-				width: 2 * map-gets($vars, input, width);
-				.tag {
-					cursor: pointer;
+			&.ng-open {
+				> input {
+					color: transparent !important;
 				}
 			}
-			&.ng-open input {
-				color: transparent !important;
+			.inputs {
+				@include display-flex();
+				@include flex-wrap(nowrap);
+				position: absolute;
+				z-index: 1;
+				top: 0;
+				bottom: 0;
+				left: 0;
+				right: 0;
+
+				.icon {
+					position: absolute;
+					top: 50%;
+					left: 50%;
+					transform: translate(-50%, -50%);
+					font-size: 0.5em;
+				}
+
+				> input {
+					width: 50% !important;
+					box-sizing: border-box;
+					border: none;
+					margin: 0;
+					outline: 0;
+					outline: none;
+					&:first-child {
```

Comment:
Now that I think about it, I'd rather use `&:first-of-type` here. Nitpicking, I know, but it's semantically better.